spec/models/bet_spec.rb

```diff
@@ -12,28 +12,31 @@
   end
 
   let(:bet1) do
-    Bet.create(bet: 'Superbowl: Atlanta vs New England, place your bets!',
+    Bet.create(bet_description: 'Superbowl: Atlanta vs New England,
+               place your bets!',
                expires_at: DateTime.tomorrow, creator_id: bob.id)
   end
-  let(:option1) { BetOption.create(option_text: 'Atlanta wins', bet: bet1) }
+  let(:option1) do
+    BetOption.create(option_text: 'Atlanta wins', bet: bet1)
+  end
   let(:option2) do
     BetOption.create(option_text: 'New England wins', bet: bet1)
   end
   let(:bob_bets_100_on_option1) do
-    UserBet.create(user:       bob,
-                   bet:        bet1,
+    UserBet.create(user: bob,
+                   bet: bet1,
```

Comment:
Huh, weird, all these spaces shouldn't have existed in the first place, so I guess all this will get resolved with I merge it with all the previous stuff. Thanks for catching it though!